doc/system_requirements.md

```diff
@@ -0,0 +1,350 @@
+<head><link href="oft_spec.css" rel="stylesheet"></head>
+
+# System Requirement Specification Exasol Row Level Security
+
+## Introduction
+
+Exasol Row Level Security (short "RLS") is a plug-in for Exasol that provides the ability to grant access on individual rows based on contents of the database.
+
+## About This Document
+
+### Target Audience
+
+The target audience are end-users, requirement engineers, software designers and quality assurance. See section ["Stakeholders"](#stakeholders) for more details.
+
+### Goal
+
+The RLS main goal is to provide fine-grained access control below the level of tables.
+
+### Quality Goals
+
+RLS's main quality goals are in descending order of importance:
+
+#### Reliable Security
+`qg~reliable-security~1`
+
+RLS reliably secures the access to protected data.
+
+<!-- Needs: qr -->
+
+#### Affordable Performance Hit
+`qg~affordable-performance-hit~1`
+
+The Performance degradation caused by and RLS-protected query compared to the same query without RLS is small.
+
+Needs: qr
+
+## Stakeholders
+
+### Data Owners
+
+Data Owners are have full access to the data _before_ row-level security is applied. They also get to decide who is allowed to read it through RLS.
+
+### Regular Users
+
+Regular Users in the context of this project are consumers of row-level security-protected data.
+
+### Terms and Abbreviations
+
+The following list gives you an overview of terms and abbreviations commonly used in OFT documents.
+
+* Column: An attribute of a defined type shared by all datasets in a table.
+* Row: Dataset in the database
+
+In the following subsections central terms are explained in more detail.
+
+## Features
+
+Features are the highest level requirements in this document that describe the main functionality of RLS.
+
+### Row Level Security
+`feat~row-level-security~1`
+
+RLS lets administrators grant access to individual table rows based on configurable criteria.
+
+Needs: req
+
+## Functional Requirements
+
+### Row Level Security With Roles
+
+#### Assigning Roles to Users
+`req~assigning-roles-to-users~1`
+
+Data Owners can assign between zero and 63 roles to users.
```

Comment:
```suggestion
Data Owners can assign roles between zero and 63 to users.
```

Reply:
This is meant to be the total amount, not the internal IDs.